PREREQUISITES.md

```diff
@@ -31,11 +31,35 @@ WP-CLI version:	1.4.1
 
 Update, if needed, to the latest stable release with `wp cli update`.
 
+#### Setting defaults for the wp valet command
+As with other `wp-cli` commands, you can set default attributes when running `wp valet`.
+
+Simply add the appropriate details in `~/.wp-cli/config.yml`:
+
+```
+valet new:
+	project: wp
+	# in: override - defaults to current directory
```

Comment:
The indentation is inconsistent. Please update the code block to use 2 spaces for indentation.